### PR TITLE
style(index.tsx): App fixed spacing issue in calibration modals

### PR DESCRIPTION
**Pull request by aptrons22**
# Overview
Closes #10677 

# Changelog

- Removed code from DirectionControl.tsx file and placed in index.tsx

# Review requests
Low

# Risk assessment
Low

### CODE REVIEW

File: app/src/molecules/JogControls/index.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
 
 import type { Jog, Plane, StepSize } from './types'
 import type { StyleProps } from '@opentrons/components'
+import { SPACING } from '@opentrons/components'
```
Comment (koji) [inline app/src/molecules/JogControls/index.tsx]: `SPACING` should be imported in line 4

**Comment by vabruzzo** [comment]
If I recall correctly, this pr addresses some comments @b-cooper had about adding the `marginBottom` to the `DirectionControl`. Should the reversion of that change be included here? Also, do we want to include this styling in the `JogControl` component itself, or do we want to pass it to all instances of that component that needs the spacing?

**Comment by aptrons22** [comment]
> If I recall correctly, this pr addresses some comments @b-cooper had about adding the `marginBottom` to the `DirectionControl`. Should the reversion of that change be included here? Also, do we want to include this styling in the `JogControl` component itself, or do we want to pass it to all instances of that component that needs the spacing?

File: app/src/molecules/JogControls/DirectionControl.tsx [hunk 2]
```diff
@@ -167,3 +167,5 @@ export function DirectionControl(props: DirectionControlProps): JSX.Element {
     </ControlContainer>
   )
 }
+
```
Comment (koji) [inline app/src/molecules/JogControls/DirectionControl.tsx]: For line 170 and 171
```suggestion

```

**Review by jerader — APPROVED** [review]
looks good! 🦖

**Review by koji — APPROVED** [review]
The changes look good to me.
Left a comment on a small thing.
before merging, please fix the lint error.

```zsh
npx prettier --write app/src/molecules/JogControls/DirectionControl.tsx
```

After that you can 
```zsh
make lint-js
```
The error should be gone.

**Pull request merged**